### PR TITLE
Bugfix/adding cleanup

### PR DESCRIPTION
**Pull request by hfordnathan**

### CODE REVIEW

File: README.md
Comment (ssurumpudi) [inline README.md:1]: the underscore is markdown for italic. You can revert changes to this file.

File: release/.env
Comment (ssurumpudi) [inline release/.env:1]: you can revert the changes in this file. The change is just a new line deletion

File: README.md
Comment (ssurumpudi) [inline README.md:1]: you can revert this change. The underscore marks the text as italic in the markdown syntax

File: release/csweb/Dockerfile
Comment (ssurumpudi) [inline release/csweb/Dockerfile:1]: I thought Aaron mentioned there was an issue with zlib. Are we good with it?

File: release/mysql/db-init/create-csweb-db.sql
Comment (ssurumpudi) [inline release/mysql/db-init/create-csweb-db.sql:1]: The diff shows. 

- Let's not drop table if they exist. This will cause a problem if they are running on an existing DB by mistake that will result in loss of data 
- The triggers are missing, why? 
-Just ensure all the constraints are in place

File: README.md
Comment (hfordnathan) [reply README.md:1]: Sure this is reverted.

Comment (hfordnathan) [reply README.md:1]: Sure this is reverted.

File: release/csweb/Dockerfile
Comment (hfordnathan) [reply release/csweb/Dockerfile:1]: Zlib seems to be working well, I tested it.

File: release/mysql/db-init/create-csweb-db.sql
Comment (hfordnathan) [reply release/mysql/db-init/create-csweb-db.sql:1]: Sure I am working through this now.

File: release/.env
Comment (hfordnathan) [reply release/.env:1]: Sure no problem, this is reverted.

File: release/mysql/db-init/create-csweb-db.sql
Comment (hfordnathan) [reply release/mysql/db-init/create-csweb-db.sql:1]: Please take a look now

**Review by ssurumpudi — APPROVED** [review]
changes look good
